modules/Movecraft/src/main/java/net/countercraft/movecraft/craft/CraftManager.java

```diff
@@ -93,8 +94,14 @@ public static CraftManager getInstance() {
             if (file.isFile()) {
 
                 if (file.getName().contains(".craft")) {
-                    CraftType type = new CraftType(file);
-                    craftTypes.add(type);
+                    try {
+                        CraftType type = new CraftType(file);
+                        craftTypes.add(type);
+                    }
+                    catch (CraftType.TypeNotFoundException | ScannerException e) {
+                        Movecraft.getInstance().getLogger().log(Level.SEVERE, I18nSupport.getInternationalisedString("Startup - failure to load craft type") + " " + file.getName());
+                        e.printStackTrace();
```

Comment:
Instead of printing a stack trace, maybe we should just print the exception message as part of the previous log output. Users often don't know how to read exceptions that include stack traces, and for an error that stems from user error I think prioritizing readability is important.